src/router/index.js

```diff
@@ -1,6 +1,16 @@
 import { createRouter, createWebHistory } from "vue-router";
 import Home from "@/pages/Home.vue";
 import Sentry from "@/pages/Sentry.vue";
+import Error from "@/pages/Error.vue";
+
+const allowedQueryParams = [
+  "sessionId",
+  "purpose",
+  "subPurpose",
+  "redirectTo",
+  "redirectID",
```

Comment:
Fix the inconsistency between `sessionId` and `redirectID` . `Id` is better, probably?

Reply:
This is still `redirectID`? Shouldn't it be `redirectId`?

Reply:
It's both, right? These are the list of allowed query parameters, so URLs, for now, can contain both `redirectID` or `redirectId`.

Reply:
If it's both, where is `redirectId`?

Reply:
There are two more params underneath our conversation :P

Reply:
I see only `group` here below the conversation, not `redirectId` :-/

Reply:
![Screenshot 2022-07-22 at 11 48 28 AM](https://user-images.githubusercontent.com/25496389/180375710-68b2b910-b914-4e4b-9fb1-812ce362a781.png)